src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -471,7 +473,16 @@ private static boolean isAllTokensAcceptable(AbstractCheck check) {
 
     private static void validateDefaultTokens(Configuration checkConfig, AbstractCheck check,
                                               Set<String> configTokens) {
-        if (Arrays.equals(check.getDefaultTokens(), check.getRequiredTokens())) {
+
+        final Set<Integer> defaultTokensSet = IntStream.of(check.getDefaultTokens())
+            .boxed()
+            .collect(Collectors.toSet());
+
+        final Set<Integer> requiredTokensSet = IntStream.of(check.getRequiredTokens())
+            .boxed()
+            .collect(Collectors.toSet());
+
+        if (defaultTokensSet.equals(requiredTokensSet)) {
```

Comment:
all changes in this file resolve `AllChecksTest.testAllCheckTokensAreReferencedInGoogleConfigFile:394->validateAllCheckTokensAreReferencedInConfigFile:451->validateDefaultTokens:479 All default tokens should be used in config for Indentation`